### PR TITLE
[quest] Crystal War: Seeing Spots, Sandora: Atelloune's Lament

### PR DESCRIPTION
**Pull request by zach2good**
Thought I was writing new content, they were there the whole time ;_;

**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] that I agree to LandSandBoat's [Limited Contributor License Agreement](https://github.com/LandSandBoat/server/blob/base/.github/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I have **read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)**
- [x] that I've _**tested my code and things my code changed**_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: scripts/quests/crystalWar/Seeing_Spots.lua [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                end,
+            },
+        },
+    },
+
+    -- Trade the wings to Wyatt to receive your reward.
+    {
+        check = function(player, status, vars)
+            return status == QUEST_ACCEPTED or status == QUEST_COMPLETED
+        end,
```
Comment (claywar) [inline scripts/quests/crystalWar/Seeing_Spots.lua:49]: Nice repeatable pattern 👍 
(See below for concern)

File: scripts/quests/crystalWar/Seeing_Spots.lua [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                    then
+                        return quest:progressEvent(4)
+                    end
+                end,
+            },
+
+            onEventFinish = {
+                [4] = function(player, csid, option, npc)
+                    player:confirmTrade()
+                end,
```
Comment (claywar) [inline scripts/quests/crystalWar/Seeing_Spots.lua:73]: Should there be a `quest:complete(player)` here? Or is this quest always logged regardless of completion?

File: scripts/quests/crystalWar/Seeing_Spots.lua [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                        })
+                    then
+                        return quest:progressEvent(4)
+                    end
+                end,
+            },
+
+            onEventFinish = {
+                [4] = function(player, csid, option, npc)
+                    player:confirmTrade()
```
Comment (Xaver-DaRed) [inline scripts/quests/crystalWar/Seeing_Spots.lua:71]: As per conversation, reminder, it isn't actually completing the quest.

Comment (zach2good) [reply scripts/quests/crystalWar/Seeing_Spots.lua:71]: Done :)

**Review by Xaver-DaRed — APPROVED** [review]

**Pull request merged**